### PR TITLE
also start of non-blinking eye icon

### PR DESCRIPTION
**Pull request by sijskes**

### CODE REVIEW

File: applet/src/os.c [hunk 1]
```diff
@@ -101,6 +101,7 @@ void * OSMboxPend_hook(OS_EVENT *pevent, uint32_t timeout, int8_t *perr)
                 last_radio_event = *(uint8_t*)ret ;
             }
             if( ((uint32_t)pevent) == 0x20017390 ) {
+                // beep events?
```
Comment (aeickho) [inline applet/src/os.c]: i have located the mbox pointer on create mbox and verified   on the calling function

Comment (sijskes) [inline applet/src/os.c]: so, i can remove the '?'

Comment (aeickho) [inline applet/src/os.c]: i have not verified it on d13.020 ...

Comment (aeickho) [inline applet/src/os.c]: and i can't check it today

Comment (sijskes) [inline applet/src/os.c]: no problem. these are d13.020 adresses. any problem here will not be fatal.

**Comment by g738l** [comment]
no redraw label "Volume" under eye icon(e38ad8eec98065b665403819913cc2428f92f4dc)


eyeicon

**Pull request merged**